web/src/app/services/IntegrationKeyCreateDialog.js

```diff
@@ -32,71 +32,69 @@ const query = gql`
   }
 `
 
-export default class IntegrationKeyCreateDialog extends React.PureComponent {
-  static propTypes = {
-    serviceID: p.string.isRequired,
-    onClose: p.func,
-  }
-
-  state = {
+export default function IntegrationKeyCreateDialog() {
+  const [state, setState] = useState({
```

Comment:
break out `value` and `errors` to separate `useState` calls rather than one object

`const [value, setValue] = useState(`
`const [errors, setErrors] = useState([])`

Reply:
Is it okay to remove the **errors** entirely because it is not used in this file and may cause linting problem?